crates/vsock/README.md

```diff
@@ -46,11 +46,11 @@ vhost-user-vsock --guest-cid=4 --uds-path=/tmp/vm4.vsock --socket=/tmp/vhost4.so
 Run qemu:
 
 ```
-qemu-system-x86_64 -drive file=/path/to/disk.qcow2 -enable-kvm -m 512M \
-  -smp 2 -vga virtio -chardev socket,id=char0,reconnect=0,path=/tmp/vhost4.socket \
-  -device vhost-user-vsock-pci,chardev=char0 \
-  -object memory-backend-file,share=on,id=mem,size="512M",mem-path="/dev/hugepages" \
-  -numa node,memdev=mem -mem-prealloc
+qemu-system-x86_64 -drive file=/path/to/disk.qcow2 -smp 2 \
```

Comment:
It might be easier to separate all the non vsock specific bits in the example to avoid confusion, e.g.:

```
qemu-system-x86_64 \
  ... normal machine options ... \
  -object memory-backend-file,share=on,id=mem0,size=512M,mem-path="/dev/hugepages" \ # size == -m size
  -chardev socket,id=char0,reconnect=0,path=/tmp/vhost4.socket \
  -device vhost-user-vsock-pci,chardev=char0
```

possibly follow with a fully working example?

Reply:
Good point, I'll update it!